src/module.ts

```diff
@@ -0,0 +1,41 @@
+import { semver } from "../deps.ts";
+
+export interface IModule {
+  name: string;
+  owner: string;
+  description: string;
+  latestVersion?: string;
+  latestStableVersion?: string;
+  packageUploadNames: string[];
+}
+
+export class Module implements IModule {
+  name: string;
+  owner: string;
+  description: string;
+  latestVersion?: string;
+  latestStableVersion?: string;
+  packageUploadNames: string[];
+
+  constructor(module: IModule) {
+    this.name = module.name;
+    this.owner = module.owner;
+    this.description = module.description;
+    this.latestVersion = module.latestVersion;
+    this.latestStableVersion = module.latestStableVersion;
+    this.packageUploadNames = module.packageUploadNames;
+  }
+
+  getLatestVersion(stable: boolean): string {
```

Comment:
```suggestion
  getLatestVersion(stable?: boolean): string {
```
Without this I was getting the following:
```ts
error: TS2345 [ERROR]: Argument of type 'boolean | undefined' is not assignable to parameter of type 'boolean'.
  Type 'undefined' is not assignable to type 'boolean'.
    latest = existing.getLatestVersion(egg.stable);
                                       ~~~~~~~~~~
    at file:///P:/GitHub/eggs/src/commands/publish.ts:181:40
```